src/components/Weazee/Weazee.module.scss

```diff
@@ -22,14 +24,23 @@
     "day day day properties"
     "forecasts forecasts forecasts forecasts";
   grid-template-rows: 1fr 340px;
-  height: 100%;
+  height: calc(100vh - #{$content-space * 2});
```

Comment:
why we need this height?

Reply:
because we need to have padding for weazeeContent block, but when we use it without height we have problem with empty space in the bottom-right corner.

Reply:
![image](https://user-images.githubusercontent.com/7966684/146285419-326d2b7d-1294-4016-bf6d-2c53e63cf1d2.png)
I don't understand how height fixes some padding issues?
What I have if I disable height(screenshot)

Reply:
This is firstly
But secondly we can't use padding for whole weazyy content(cause of some UI feature in weazee forecasts we mustn't use any horizontal padding(left/right) for weazeeContent and weazee root